### PR TITLE
Update openSUSE and SUSE package availability

### PR DESCRIPTION
**Pull request by AdamMajer**

### CODE REVIEW

**Review by richardlau — APPROVED** [review]

File: locale/en/download/package-manager.md [hunk 1]
```diff
@@ -301,18 +301,22 @@ pkg_add node
...
+For example, to install Node.js 14.x on openSUSE Leap 15.2, run the following as root:
 
 ```bash
-zypper install nodejs4
+zypper install nodejs14
 ```
 
+Different major versions of Node can be installed and used concurrently.
+
+
```
Comment (nschonni) [inline locale/en/download/package-manager.md]:
```suggestion
```

Comment (AdamMajer) [reply locale/en/download/package-manager.md]: Fixed now. Forgot to run the test suite :cry:

Comment (nschonni) [reply locale/en/download/package-manager.md]: No worries, thanks for fixing

File: locale/en/download/package-manager.md [hunk 2]
```diff
@@ -301,18 +301,21 @@ pkg_add node
 
 Node.js is available in the main repositories under the following packages:
 
-* **openSUSE Leap 42.2**: `nodejs4`
-* **openSUSE Leap 42.3**: `nodejs4`, `nodejs6`
-* **openSUSE Tumbleweed**: `nodejs4`, `nodejs6`, `nodejs8`
-* **SUSE Linux Enterprise Server (SLES) 12**: `nodejs4`, `nodejs6`
-  (The "Web and Scripting Module" must be [added before installing](https://www.suse.com/documentation/sles-12/book_sle_deployment/data/sec_add-ons_extensions.html).)
+* **openSUSE Leap 15.2**: `nodejs10`, `nodejs12`, `nodejs14`
```
Comment (nschonni) [inline locale/en/download/package-manager.md:304]: Just curious how this when from 42 to 15? Haven't really used SUSE, so I'm guessing this is right, but it seems odd

Comment (AdamMajer) [reply locale/en/download/package-manager.md:304]: Leap 42 was based on SLE 12. Then there was some discussion and the next version was deemed SLE 15. openSUSE was then renumbered to match. While previously the sources of core packages were shared, now the binaries are shared too to allow users to upgrade to SLES from Leap without reinstalling anything.

So yes, the numbers changed funnily. For openSUSE 13 -> 42 -> 15.

**Pull request merged**